generated/formats/dds/enum/D3D10ResourceDimension.py

```diff
@@ -7,8 +7,8 @@ class D3D10ResourceDimension(UintEnum):
 	"""
 	An unsigned 32-bit integer. Identifies the type of resource being used.
 	"""
-	D3D10_RESOURCE_DIMENSION_UNKNOWN = 0
-	D3D10_RESOURCE_DIMENSION_BUFFER = 1
-	D3D10_RESOURCE_DIMENSION_TEXTURE1D = 2
-	D3D10_RESOURCE_DIMENSION_TEXTURE2D = 3
-	D3D10_RESOURCE_DIMENSION_TEXTURE3D = 4
+	D_3_D_10_RESOURCE_DIMENSION_UNKNOWN = 0
```

Comment:
Nope, looks really bad

Reply:
This was changed two commits after that, now the enum key formatting is only applied to enum keys with whitespace or lower case letters, so you don't get that.